src/common.jl

```diff
@@ -11,3 +11,11 @@ end
 struct InvalidValue <: Exception
     msg::AbstractString
 end
+
+struct ConflictingInputs <: Exception
+    msg::AbstractString
+end
+
+struct HashMismatch <: Exception
+    msg::AbstractString
+end
```

Comment:
There is a longstanding convention that each file should end in a newline.  Git complains if you don't have it.  Looks like you took it away here.  Can you add it back?  If you're using VSCode then you can do this automatically with `files.insertFinalNewline": true` in your config settings.